backend/src/main/java/com/benine/backend/ServerController.java

```diff
@@ -64,9 +65,14 @@ public static synchronized ServerController getInstance() {
    */
   public void start() {
     httpController = new HttpController(config.getValue("serverip"),
-        Integer.parseInt(config.getValue("serverport")), logger); 
-    
+        Integer.parseInt(config.getValue("serverport")), logger);
     startupDatabase();
+    try {
```

Comment:
If this try block fails, the application becomes useless because there is no data. Maybe we should throw an exception here to notify the main about a start() failure.